src/shared/components/post/post-form.tsx

```diff
@@ -269,6 +270,17 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
             {url && isImage(url) && (
               <img src={url} className="img-fluid" alt="" />
             )}
+            {this.state.imageDeleteUrl && (
```

Comment:
Using `this.state.imageDeleteUrl` for visibility of this button ensures it only appears alongside uploaded images.